### ISSUE TITLE
`listsinceblock` transaction format differs from bitcoin core's

### ISSUE THREAD

**Issue by arnuschky** [btcsuite/btcwallet#264]
Bitcoin core lists `blockindex` and `vout`, while btcd lists `fee` and `otheraccount`. Outputs below (printed python dict rather than json, sorry for that).

```
{u'lastblock': u'000000000d54b5ad2be8b327aeccc522d56a6447d3b129e5e3986bb9398ee121',
 u'transactions': [{u'account': u'',
                    u'address': u'migiScBNvVKYwEiCFhgBNGtZ87cdygtuSQ',
                    u'amount': Decimal('0.01000000'),
                    u'blockhash': u'000000000d54b5ad2be8b327aeccc522d56a6447d3b129e5e3986bb9398ee121',
                    u'blockindex': 19,
                    u'blocktime': 1430861682,
                    u'category': u'receive',
                    u'confirmations': 1,
                    u'time': 1430854362,
                    u'timereceived': 1430854362,
                    u'txid': u'ebf22d13f04aa8e6c91ff495455a32815c4967253f232653973cc4bce4622fe0',
                    u'vout': 0,
                    u'walletconflicts': []}]}
```

```
{u'lastblock': u'000000000d54b5ad2be8b327aeccc522d56a6447d3b129e5e3986bb9398ee121',
 u'transactions': [{u'account': u'',
                    u'address': u'migiScBNvVKYwEiCFhgBNGtZ87cdygtuSQ',
                    u'amount': Decimal('0.01000000'),
                    u'blockhash': u'000000000d54b5ad2be8b327aeccc522d56a6447d3b129e5e3986bb9398ee121',
                    u'blocktime': 1430861682,
                    u'category': u'receive',
                    u'confirmations': 1,
                    u'fee': 0,
                    u'otheraccount': u'',
                    u'time': 1430854362,
                    u'timereceived': 1430854362,
                    u'txid': u'ebf22d13f04aa8e6c91ff495455a32815c4967253f232653973cc4bce4622fe0',
                    u'walletconflicts': []}]}
```

**Comment by jrick** [comment]
I have this fixed locally but we'll be releasing the next wallet version before I push it out since it requires changes in the btcjson package and I don't want a wallet release depending on an unreleased version of btcd.

**Comment by jrick** [comment]
On second thought I can PR these now and we'll release wallet later (in a week or less?) along with a point btcd release.

**Comment by arnuschky** [comment]
Ok, great. :)

**Comment by jrick** [comment]
fixed by #265
